docs/en_US/Compression/QuickStart.md

```diff
@@ -193,4 +195,18 @@ pruner.export_model(model_path='model.pth')
 pruner.export_model(model_path='model.pth', mask_path='mask.pth', onnx_path='model.onnx', input_shape=[1, 1, 28, 28])
 ```
 
+#### Export Quantized Model
+
+You can export the quantized model directly by using ```torch.save``` api and the quantized model can be loaded by ```torch.load``` without any extra modification. The following example shows the normal proceduce of saving and loading quantized model in QAT.
```

Comment:
better to briefly explain where is quantization related data saved, for example, scale, zero point etc.

Reply:
Agree.